taiyaki/basecall_helpers.py

```diff
@@ -8,13 +8,6 @@
 _DEFAULT_OVERLAP = 100
 
 
-def round_chunk_values(chunk_size, chunk_overlap, stride):
```

Comment:
It's used below in `run_model` which I think is where @marcus1487 hooks into taiyaki?

Reply:
Not any more .  I can’t easily grep the code base for other occurrences from here.